modules/concept-docs/pages/n1ql-query.adoc

```diff
@@ -22,15 +22,15 @@ Do not turn off the `adhoc` flag for _every_ query to Server 6.0 and earlier, si
 include::example$n1ql-concept.php[tag=prepared-statement]
 ----
 
-CAUTION: *When running an application using Prepared Statements through the PHP SDK* -- if you plan to upgrade Couchbase Server from 6.0.x or earlier to 6.5.0 or later, and are running a version of the PHP SDK with an underlying LCB prior to 2.10.6, you will need to xref:7.0@server:install:upgrade-strategy-for-features.adoc#prepared-statements[restart the app or otherwise work around] a change in the Server's behaviour.
+CAUTION: *When running an application using Prepared Statements through the PHP SDK* -- if you plan to upgrade Couchbase Server from 6.0.x or earlier to 6.5.0 or later, and are running a version of the PHP SDK with an underlying LCB prior to 2.10.6, you will need to xref:7.1@server:install:upgrade-strategy-for-features.adoc#prepared-statements[restart the app or otherwise work around] a change in the Server's behaviour.
```

Comment:
Worth checking if we can remove this from here, and possibly the previous two doc versions?

Reply:
Check with PM?
6.0 is still in the compat page, and 2->3 migration is still covered, so unsure what criterion is for removing this admonition?

Reply:
Technically, you could probably be "running a version of the PHP SDK with an underlying LCB prior to 2.10.6" with PHP SDK 3.0.0,
but I don't think it's going to happen with SDK 3.1 - 4.0